### DISCUSSION TITLE
Clean Installation of AirSim with PX4 and ROS2 & Custom Aircraft

### DISCUSSION THREAD

**Discussion by JulianDev24** [microsoft/AirSim#4885] [Support Q&A]
Hey guys, I'm new to all the simulation stuff and I have some questions. 

What's the best way to cleanly install AirSim along with ROS2 and PX4 on Ubuntu 22.04? 
And could you provide some guidance on how to integrate my own aircraft (a drone) into AirSim and equip it with multiple cameras and other sensors, with the goal of effectively reading sensor data in ROS2? 

Thanks in advance for your help!

**Comment by mzahana** [comment]
Hi @JulianDev24 . Where you able to figure this out?
